### DISCUSSION TITLE
Is it possible to sync light/dark with the os theme?

### DISCUSSION THREAD

**Discussion by martisj** [mcchrish/zenbones.nvim#120] [Q&A] [answered]
I have got wezterm to respond to the os theme but that doesn't seem to apply to neovim with LazyVim and Neobones theme.

Is there a way to get `vim.o.background` to sync with wezterms bg?

**Comment by evantravers** [comment, marked as answer]
Yes it is! https://evantravers.com/articles/2023/04/04/wezterm-and-neovim-dark-mode-synced-to-osx/
